pkg/api/validation/validation_test.go

```diff
@@ -2892,20 +2892,20 @@ func TestValidateLimitRange(t *testing.T) {
 			{
 				Type: api.LimitTypePod,
 				Max: api.ResourceList{
-					api.ResourceCPU:    resource.MustParse("100"),
-					api.ResourceMemory: resource.MustParse("10000"),
+					api.ResourceCPU:    resource.MustParse("100m"),
+					api.ResourceMemory: resource.MustParse("10000Mi"),
 				},
 				Min: api.ResourceList{
-					api.ResourceCPU:    resource.MustParse("5"),
-					api.ResourceMemory: resource.MustParse("100"),
+					api.ResourceCPU:    resource.MustParse("5m"),
+					api.ResourceMemory: resource.MustParse("100Mi"),
```

Comment:
You could make this test much denser (thus easier to read) by making a `func RL(cpu, mem string) api.ResourceList`